api/tests/integration/domain/usecases/create-organizations-with-tags-and-target-profiles_test.js

```diff
@@ -36,361 +36,378 @@ describe('Integration | UseCases | create-organizations-with-tags-and-target-pro
-        {
-          attribute: 'type',
-          message: 'Le type n’est pas renseigné.',
-        },
-        {
-          attribute: 'externalId',
-          message: "L'externalId n’est pas renseigné.",
-        },
-        {
-          attribute: 'name',
-          message: 'Le nom n’est pas renseigné.',
-        },
-        {
-          attribute: 'tags',
-          message: 'Les tags ne sont pas renseignés.',
-        },
-        {
-          attribute: 'locale',
-          message: "La locale doit avoir l'une des valeurs suivantes : fr-fr, fr ou en",
-        },
-        {
-          attribute: 'locale',
-          message: "La locale n'est pas renseignée.",
-        },
-        {
-          attribute: 'credit',
-          message: 'Le crédit doit être un entier.',
-        },
-        {
-          attribute: 'createdBy',
-          message: "L'id du créateur doit être un nombre",
-        },
-      ]);
     });
 
-    it('should throw an error when first error found when an externalId is missing and email is not valid', async function () {
-      //given
-      const organizationsWithTagsWithOneMissingExternalId = [
-        {
-          type: 'PRO',
-          externalId: 'b200',
-          name: 'Youness et Fils',
-          provinceCode: '123',
-          credit: 0,
-          emailInvitations: 'youness',
-          locale: 'fr-fr',
-          tags: 'TagNotFound',
-          createdBy: userId,
-          documentationUrl: 'http://www.pix.fr',
-          targetProfiles: '1_2_3',
-          organizationInvitationRole: 'ADMIN',
-        },
-        {
-          type: 'PRO',
-          externalId: '',
-          name: 'Andreia & Co',
-          provinceCode: '345',
-          credit: 10,
-          emailInvitations: 'andreia@example.net',
-          locale: 'fr-fr',
-          tags: 'TagNotFound',
-          createdBy: userId,
-          documentationUrl: 'http://www.pix.fr',
-          targetProfiles: '1_2_3',
-          organizationInvitationRole: 'ADMIN',
-        },
-        {
-          type: 'PRO',
-          externalId: 'b201',
-          name: 'Mathieu Bâtiment',
-          provinceCode: '567',
-          credit: 20,
-          emailInvitations: 'mathieu@example.net',
-          locale: 'fr-fr',
-          tags: 'TagNotFound',
-          createdBy: userId,
-          documentationUrl: 'http://www.pix.fr',
-          targetProfiles: '1_2_3',
-          organizationInvitationRole: 'ADMIN',
-        },
-      ];
-
-      // when
-      const error = await catchErr(createOrganizationsWithTagsAndTargetProfiles)({
-        domainTransaction,
-        organizations: organizationsWithTagsWithOneMissingExternalId,
-        organizationRepository,
-        tagRepository,
-        targetProfileShareRepository,
-        organizationTagRepository,
-        organizationInvitationRepository,
-        dataProtectionOfficerRepository,
+    context('when first error found when an externalId is missing and email is not valid', function () {
```

Comment:
Je ne comprends pas bien ce message de contexte mais vu le `when` j'ai l'impression que ça devrait être :

```suggestion
    context('when email is not valid', function () {
```